libs/chonky/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+cargo-features = ["edition2024"]
+
+[package]
+name = "chonky"
+version = "0.0.0"
+publish = false
+authors = ["Jesus Fileto <jesus.fileto@yale.edu>", "HASH"]
+edition.workspace = true
+license.workspace = true
+
+[dependencies]
+# Public workspace dependencies
+error-stack = { workspace = true, public = true }
```

Comment:
`error-stack` is the error-library we built at HASH. If we do error-handling we favor error-stack over other error-libraries (such as `eyre` or `anyhow`).

Reply:
Docs for `error-stack`: https://docs.rs/error-stack/0.5.0/error_stack/

Reply:
For the up-to-date documentations for `error-stack` (we use the in-tree library) you need to generate the docs using `cargo doc`.